src/langsmith/user-management.mdx

```diff
@@ -254,6 +255,30 @@ For additional information, see Okta's [documentation](https://help.okta.com/en-
 
 **Step 1: Create and configure the Okta SAML application**
 
+**Via Okta Integration Network (recommended)**
+
+1. Sign in to [Okta](https://login.okta.com/).
+1. In the upper-right corner, select Admin. The button is not visible from the Admin area.
```

Comment:
Not sure if we need any styling for things like Admin, Add Integration, etc. in these steps